go.mod

```diff
@@ -7,27 +7,28 @@ require (
 	contrib.go.opencensus.io/exporter/prometheus v0.1.1-0.20191218042359-6151c48ac7fa
 	github.com/cilium/ebpf v0.0.0-20200317182658-6f632cc9ee38
 	github.com/evanphx/json-patch v4.2.0+incompatible
-	github.com/golang/protobuf v1.3.2
-	github.com/google/go-cmp v0.3.1
+	github.com/golang/protobuf v1.4.3
+	github.com/google/go-cmp v0.4.0
 	github.com/hashicorp/go-multierror v1.0.0
 	github.com/intel/cri-resource-manager/pkg/topology v0.0.0
+	github.com/marquiz/goresctrl v0.0.0-20201110110642-51d677608af3
```

Comment:
Let's update this to `github.com/marquiz/goresctrl v0.0.0-20201113093610-2b83b62c0854`. After that we could merge this in principle IMO.